client/components/SearchResult.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+
+export default class SearchResult extends React.Component {
+
+  static get propTypes () {
+    return {
+      myData: React.PropTypes.object

```

Comment:
I don't know. I have been working with classes now for years. So I find it more confusing to have this script like syntax.
And putting the export default somwhere else did not work